### DISCUSSION TITLE
GitHub actions macOS 12 runner network issue (presumably) with plain Kubernetes

### DISCUSSION THREAD

**Discussion by saschagrunert** [lima-vm/lima#1983] [Q&A]
### Description

Hey folks, I'm planning to switch from Vagrant (VirtualBox) to lima within our GitHub actions based e2e tests of the Security Profiles Operator: https://github.com/kubernetes-sigs/security-profiles-operator/pull/1953

It works seamlessly on Linux, but the macOS 12 runner seems to block basic network connections like simple health checks to `http://10.85.0.11:8085/healthz`: https://github.com/kubernetes-sigs/security-profiles-operator/actions/runs/6734913889/job/18307021832?pr=1953

**Is there anything additional I have to configure to make the pod network work on mac OS?**

Configuration YAML: https://github.com/kubernetes-sigs/security-profiles-operator/blob/d0b1d80333f7a678c04700cbcad8853a9e1d2cc5/hack/ci/lima-ubuntu.yaml

cc @afbjorklund

**Comment by afbjorklund** [comment]
If I remember correctly, this is using the "bridge" CNI configuration that is bundled with the `cri-o`package

https://github.com/cri-o/packaging/blob/main/templates/latest/cri-o/bundle/11-crio-ipv4-bridge.conflist
